src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -285,6 +285,18 @@ protected void wrapUp(GHTag[] page) {
         };
     }
 
+    /**
+     * List languages for the specified repository.
+     * The value on the right of a language is the number of bytes of code written in that language.
+     * {
+         "C": 78769,
+         "Python": 7769
+       }
+     */
+    public Map<String,Integer> listLanguages() throws IOException {
```

Comment:
Size in bytes, max Integer will be 2Gb, i think some crazy repo may have such number. Long looks better.